plugins/additional/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_library(additional SHARED additional.cpp)
+
+target_link_libraries(additional
+    ticpp
+    wxfb-plugin-interface
+)
+
+if(MSYS)
+    install(FILES
+        ${CMAKE_CURRENT_BINARY_DIR}/libadditional${CMAKE_SHARED_LIBRARY_SUFFIX}
+        DESTINATION ${INSTALL_LIB_DIR}/plugins/additional/
+    )
+elseif(UNIX)
+    install(FILES
+        ${CMAKE_CURRENT_BINARY_DIR}/libadditional${CMAKE_SHARED_LIBRARY_SUFFIX}
+        DESTINATION ${INSTALL_LIB_DIR}
+    )
+endif()
```

Comment:
Why differentiate between the platforms, can't the installation directory propertly set for each platform?
Why is the output library name hardcoded, can't it be queried from the target?

Reply:
Code replaced by function. I added these paths based on where wxFormBuilder looks for plugins.